src/people/widgetViews/workspace/style.ts

```diff
@@ -1389,20 +1396,17 @@ export const GenerateStoriesTitle = styled.h2`
 `;
 
 export const GenerateStoriesContent = styled.div`
-  padding: 40px 32px;
-  min-height: 200px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
+  padding: 40px 70px;
+  flex: 1;
+  overflow-y: auto;
 `;
 
 export const GenerateStoriesText = styled.p`
   font-family: 'Barlow', sans-serif;
   font-size: 18px;
   color: #5f6368;
   margin: 0;
-  width: 500px;
+  width: 650px;

```

Comment:
Here also, can we use a width of `100%` so the content just take the width of it's container. Using absolute width always caused issue because users always have different screen size